pymatgen/analysis/chemenv/coordination_environments/coordination_geometries.py

```diff
@@ -908,7 +908,7 @@ def __init__(self, permutations_safe_override=False, only_symbols=None):
 
         self.minpoints = {}
         self.maxpoints = {}
-        self.separations_cg = {}
+        self.separations_cg: dict = {}
```

Comment:
can we make the types more specific? `list`/`dict`/`set`/... alone don't add any information for the reader. `dict[str, CoordinationGeometry]`, `list[Structure]`, etc. is more helpful

Reply:
> can we make the types more specific? `list`/`dict`/`set`/... alone don't add any information for the reader. `dict[str, CoordinationGeometry]`, `list[Structure]`, etc. is more helpful

Yes this is very true. But in this PR I was working on adding types for `io.vasp`, sometimes `mypy` would complain about other modules (`need annotation for some line`) which I haven't yet got time to look closer into (you know the situation, when there are hundreds of errors to fix). But I would keep this in mind and try to more specific types :)